modules/core/src/smithy4s/schema/Schema.scala

```diff
@@ -90,8 +90,11 @@ sealed trait Schema[A]{
   final def biject[B](to: A => B)(from: B => A) : Schema[B] = Schema.bijection(this, to, from)
   final def option: Schema[Option[A]] = Schema.option(this)
 
+  final def nullable: Schema[Nullable[A]] = Nullable.schema(this)
+
   final def isOption: Boolean = this match {
     case _: OptionSchema[_] => true
+    case BijectionSchema(underlying, _) => underlying.isOption // TODO AJ: does this have any side-effects in the other places .isOption is used? is it a breaking change?
```

Comment:
See TODO comment...

This is necessary in order to get the new code in `DocumentDecoderSchemaVisitor` (which I copied from the original PR) to work for `Nullable.schema`, but I'm not sure if it breaks anything else. The main alternatives I can see are to drop that check and just always delegate null values to the underlying decoder if a `nullable` hint is present (is that safe?), or define the new schema here instead of as a bijection from option.